File: spring-data-requery-java/src/test/java/com/coupang/springframework/data/requery/java/repository/sample/ProductRepository.java
package com.coupang.springframework.data.requery.java.repository.sample;

import com.coupang.springframework.data.requery.java.domain.sample.Product;
import com.coupang.springframework.data.requery.repository.RequeryRepository;

/**
 * ProductRepository
 *
 * @author debop@coupang.com
 * @since 18. 6. 14
 */
public interface ProductRepository extends RequeryRepository<Product, Long> {
}

File: spring-data-requery-kotlin/src/test/kotlin/org/springframework/data/requery/kotlin/repository/sample/ProductRepository.kt
package org.springframework.data.requery.kotlin.repository.sample

import org.springframework.data.requery.kotlin.domain.sample.Product
import org.springframework.data.requery.kotlin.repository.RequeryRepository

/**
 * org.springframework.data.requery.repository.sample.ProductRepository
 *
 * @author debop
 */
interface ProductRepository: RequeryRepository<Product, Long> {
}